init.go

```diff
@@ -197,15 +197,29 @@ func (a *initFnCmd) bindRoute(rt *models.Route) {
 	}
 }
 
+// validateFuncName checks if the func name is valid, the name can't contain a colon and
+// must be all lowercase
+func validateFuncName(name string) error {
```

Comment:
this doesn't solve a problem completely, take a look at this https://github.com/fnproject/fn/blob/master/api/models/app.go#L30, the colon is not the only invalid char for the app name

Reply:
Hi Denis thanks for the review.
I am not sure I get your concern, I am trying to fix this issue: https://github.com/fnproject/cli/issues/103 which regards the name of the docker image created, the code you pointed me out refers to the validation of App name, am I right?
The validation rules for the App name are not a strict constraints for image name, for example I can have an image name longer than 30 chars.

Reply:
Indeed, uppercase names is not the only one possible case when an app name is considered as invalid. Okay,
I see, had wrong context. LGTM.